cts/cts-scheduler.in

```diff
@@ -1293,7 +1293,7 @@ class CtsScheduler(object):
 
         # (Intermediate) test outputs
         output_filename = "%s/%s.out" % (self.args.out_dir, test_name)
-        dot_output_filename = "%s/%s.pe.dot" % (self.args.out_dir, test_name)
+        dot_output_filename = "%s/%s.dot.pe" % (self.args.out_dir, test_name)
```

Comment:
This has annoyed me for awhile but I never bothered to change it :)

Reply:
I only did it because I noticed that the `clean-local` glob would match only the dot pe files and none of the other ones. That didn't seem right :)